packages/data-point/examples/async-example.js

```diff
@@ -15,7 +15,7 @@ dataPoint.addEntities({
 
   // model entity to resolve a Planet
   'model:Planet': {
-    inputType: 'schema:DataInput',
+    // inputType: 'schema:DataInput',
```

Comment:
why is it commented out? remove it?

Reply:
yeah placing it back hehe so much...so many... so .... thanks!